newIDE/app/src/AssetStore/ExtensionStore/ExtensionInstallDialog.js

```diff
@@ -145,6 +157,20 @@ export default class ExtensionInstallDialog extends Component<Props, State> {
               <Text noMargin size="body2">
                 <Trans>Version {' ' + extensionShortHeader.version}</Trans>
               </Text>
+              <div style={styles.authors}>
```

Comment:
Use `<Line noMargin>` (Line/Column are what we use for most layouts). This avoids custom styles (like `authors`) can ensure consistency of margins + avoid the notion of "divs" in the UI.